temporaldata/data.py

```
@@ -476,6 +476,35 @@ def get_nested_attribute(self, path: str) -> Any:
+        set the value of the ``id`` attribute of the ``session`` object.
+
+        Args:
+            path: Nested attribute path (can be dot-separated, e.g. "session.id").
+            value: The value to set for the attribute.
+
+        Returns:
+            Data: self with the updated nested attribute.
+
+        Raises:
+            AttributeError: If any component of the path cannot be resolved.
+        """
```

Ruff (0.14.10)

f-string used as docstring. Python will interpret this as a joined string, rather than a docstring.

(B021)